src/js/selfVoicing.js

```diff
@@ -92,7 +92,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
 
     gpii.firstDiscovery.selfVoicing.queueSpeech = function (that, text, options) {
         if (that.model.enabled) {
-            fluid.textToSpeech.queueSpeech(that, text, true, options);
+            fluid.textToSpeech.queueSpeech(that, text, !fluid.get(options, "queue"), options);
```

Comment:
I used queue instead, because it is the inverse of the typical action. So in this case it will interrupt by default, whereas in the underlying implementation of fluid.textToSpeech.queueSpeech, it queues by default.